data/campaigns/Heir_To_The_Throne/scenarios/06_The_Siege_of_Elensefar.cfg

```diff
@@ -232,325 +232,381 @@
         [/message]
     [/event]
 
+
+
+# Thieves’ Guild
+# Handle the following cases:
+#  1) Enemies defeated before turn 4, four thieves join the revelry in town
```

Comment:
Further down I admit not unless debugging or cheating, but it completes the logic table and covers all possible cases, so I like it.